third_party/mbedtls/repo/library/entropy_poll.c

```diff
@@ -50,6 +50,28 @@
 
 #if defined(_WIN32) && !defined(EFIX64) && !defined(EFI32)
 
+#if defined(_KERNEL_MODE)
+
+#include <ntdef.h>
+#include <bcrypt.h>
+
+int mbedtls_platform_entropy_poll( void *data, unsigned char *output, size_t len,
```

Comment:
I looked at the code path that calls `otPlatSecureRandomGet` and it looks like you guys already hooked that up to the entropy, so this ended up being unnecessary now. I had added this months ago, before you made that change, and I hadn't tried to compile without it since. I will remove this.